src/stores/rootStore.ts

```diff
@@ -112,54 +113,129 @@ export const useRootStore = defineStore('root', {
+        state.bootstrapFilingStatus === FilingStatus.AWAITING_REVIEW
+      )
     },
 
-    isDraftIncorpApp (state: RootStateIF): boolean {
-      return (state.entityStatus === EntityStatus.DRAFT_INCORP_APP)
+    /** Whether this is a continuation in bootstrap filing that belongs in the Filing History List. */
+    isContinuationInFiling (state: RootStateIF): boolean {
+      return (
+        state.bootstrapFilingType === FilingTypes.CONTINUATION_IN &&
+        (
+          state.bootstrapFilingStatus === FilingStatus.APPROVED ||
+          state.bootstrapFilingStatus === FilingStatus.COMPLETED ||
+          state.bootstrapFilingStatus === FilingStatus.PAID ||
+          state.bootstrapFilingStatus === FilingStatus.REJECTED
+        )
+      )
     },
 
-    isDraftRegistration (state: RootStateIF): boolean {
-      return (state.entityStatus === EntityStatus.DRAFT_REGISTRATION)
+    /** Whether this is an incorporation application bootstrap filing that belongs in the Todo List. */
+    isIncorporationApplicationTodo (state: RootStateIF): boolean {
+      return (
+        state.bootstrapFilingType === FilingTypes.INCORPORATION_APPLICATION &&
+        (
+          state.bootstrapFilingStatus === FilingStatus.DRAFT ||
+          state.bootstrapFilingStatus === FilingStatus.PENDING
+        )
+      )
     },
 
-    isFiledAmalgamation (state: RootStateIF): boolean {
-      return (state.entityStatus === EntityStatus.FILED_AMALGAMATION)
+    /** Whether this is an incorporation application bootstrap filing that belongs in the Filing History List. */
+    isIncorporationApplicationFiling (state: RootStateIF): boolean {
+      return (
+        state.bootstrapFilingType === FilingTypes.INCORPORATION_APPLICATION &&
+        (
+          state.bootstrapFilingStatus === FilingStatus.COMPLETED ||
+          state.bootstrapFilingStatus === FilingStatus.PAID
+        )
+      )
     },
 
-    isFiledContinuationIn (state: RootStateIF): boolean {
-      return (state.entityStatus === EntityStatus.FILED_CONTINUATION_IN)
+    /** Whether this is a registration bootstrap filing that belongs in the Todo List. */
+    isRegistrationTodo (state: RootStateIF): boolean {
+      return (
+        state.bootstrapFilingType === FilingTypes.REGISTRATION &&
+        (
+          state.bootstrapFilingStatus === FilingStatus.DRAFT ||
+          state.bootstrapFilingStatus === FilingStatus.PENDING
+        )
+      )
     },
 
-    isFiledIncorpApp (state: RootStateIF): boolean {
-      return (state.entityStatus === EntityStatus.FILED_INCORP_APP)
+    /** Whether this is a registration bootstrap filing that belongs in the Filing History List. */
+    isRegistrationFiling (state: RootStateIF): boolean {
+      return (
+        state.bootstrapFilingType === FilingTypes.REGISTRATION &&
+        (
+          state.bootstrapFilingStatus === FilingStatus.COMPLETED ||
+          state.bootstrapFilingStatus === FilingStatus.PAID
+        )
+      )
```

Comment:
So this ^^ is the new logic. It's more verbose but I think it's easier to understand and it's easier to extend if needed.